test/jasmine/tests/calcdata_test.js

```diff
@@ -1160,6 +1160,23 @@ describe('calculated data and points', function() {
                         checkAggregatedValue(baseMock, expectedAgg, false, done);
                     });
 
+                    it('takes the geometric mean of all values per category across traces of type ' + trace.type, function(done) {
```

Comment:
Let's skip `ohlc` and `candlestick` which fail with tiny tolerance.
```suggestion
                    it('takes the geometric mean of all values per category across traces of type ' + trace.type, function(done) {
                        if(trace.type === 'ohlc' || trace.type === 'candlestick') return done();

```

Reply:
Sure thing.